src/components/SummaryCard/SummaryCardContainer/_mocks_/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @file Summary card container mocks.
+ * @copyright IBM Security 2019
+ */
+
+export default {
+  summaryCards: new Array(5)
+    .fill()
+    .map((title = 'summary-card', id) => ({ id: `${title}__${id}` })),
+};
```

Comment:
Related to my previous comment about the structure of this component - I hope we can avoid arrays like this. What do you think?